integration_tests/test_commit_format.py

```diff
@@ -3,12 +3,16 @@
 """Test the commit message format."""
 
 import os
+import subprocess
 
 from utils import get_cmd_output
 
 COMMIT_TITLE_MAX_LEN = 50
 COMMIT_BODY_LINE_MAX_LEN = 72
 BASE_BRANCH = os.environ['BUILDKITE_PULL_REQUEST_BASE_BRANCH']
+# When preparing the test environment, `origin` is set to the
+# current upstream repository ('BUILDKITE_REPO').
+BASE_REPO = "origin"
```

Comment:
I would prefer us to use BUILDKITE_REPO here instead of origin. `origin` is just the name, and the name can always be updated. Using the env variable sounds less prone to error to me.

Reply:
I agree, for this test seems cleaner this way. In the benchmark test on the other hand, it seems we are hardcoding `origin` (and other things too) for a nicer `git checkout` command. After `git fetch some_url master`, I think `git checkout FETCH_HEAD` could be used, but anyway, more changes to that test are required and I think the only good way to test them is in a PR that adds some benchmarks or at least a benchmark pipeline.
In the meantime, I updated the commit test.